### PR TITLE
Added documentation for ENCODE BigBed parse functions and instructions of creating custom bigbed parser functions(Address issue #2)

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Brought to you by UMass Medical School, Weng Lab.
 
 ## Installation
 
-For npm use: `npm install bigwig-reader --save`
+For npm use: `npm install bigwig-reader`
```
Comment (hpratt) [inline README.md:9]: this can be changed to genomic-reader

File: README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Brought to you by UMass Medical School, Weng Lab.
...
 
 For yarn use: `yarn add bigwig-reader`
```
Comment (hpratt) [inline README.md:11]: genomic-reader here too

**Review by hpratt — CHANGES_REQUESTED** [review]
Looks great! Thanks so much for your work getting these features integrated. I have fixed the NPM publish workflow so it should now be possible to use the NPM package name genomic-reader rather than bigwig-reader. Could you change those references on line 9 and 11? Otherwise I think the documentation is very clear.

File: README.md (hunk 2 above)
Comment (zoldello) [reply README.md]: @hpratt Done. Thanks for catching this. I fixed this.  

While I got your attention, when I do install genomic-reader with yarn or npm, I do not get the dist folder. Could it be something wrong with my local machine?

**Comment by jackh726** [comment]
This is great!

**Comment by jackh726** [comment]
Also, just fyi @zoldello, you don't *have* to create a new issue per PR! (But feel free to do so if you find an issue but can't/don't want to make a PR)

Thanks again for your work here!

File: README.md (hunk 2 above)
Comment (hpratt) [reply README.md]: hmm that was the exact issue I was trying to fix, are you forcing v1.4.7? with "genomic-reader": "1.4.7" in package.json it does have the dist folder on my machine

Comment (zoldello) [reply README.md]: @hpratt Thanks for confirming the dist issue. I was using v1.4.1. I will try try the new updates later to see if dist is automatically added. Thanks for fixing the issue.

**Comment by zoldello** [comment]
@jackh726 Acknowledged and pardon me. Going forward, I will make issues only if I cannot fix a problem myself. Otherwise, I will just make a PR. And again, thanks for your code reviews. They were extremely helpful.

**Pull request merged**